monitor.cpp

```diff
@@ -4,35 +4,66 @@
 #include <chrono>
 #include <iostream>
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
-
+/// @brief displayWarning function to display warning message
+void displayWarning() {
+  for (int i = 0; i < 6; i++) {
+    cout << "\r* " << flush;
+    sleep_for(seconds(1));
+    cout << "\r *" << flush;
+    sleep_for(seconds(1));
+  }
+}
+/// @brief to check if the vitals are ok
+/// @param temperature
+/// @param pulseRate
+/// @param spo2
+/// @return 1 if vitals are ok, 0 otherwise
```

Comment:
The purpose of `vitalsOk` is same as `checkVitals` - it's just a pass-thru function.
Is it needed?